hazelcast/src/test/java/com/hazelcast/config/properties/PropertyTypeConverterTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2008-2014, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config.properties;
+
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category(QuickTest.class)
+public class PropertyTypeConverterTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void test_string_converter_fail() throws Exception {
+        PropertyTypeConverter.STRING.convert(null);
+    }
+
+    @Test
+    public void test_string_converter_success() throws Exception {
+        assertEquals("test", PropertyTypeConverter.STRING.convert("test"));
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void test_short_converter_illegal_argument() throws Exception {
```

Comment:
In the testing we use mostly the following naming practice:

test_shortConverted_whenIllegalArgument
shortConverted_whenIllegalArgument
or 
test_shortConverted_whenIllegalFormat_thenNumberFormatException
shortConverted_whenIllegalFormat_thenNumberFormatException.